rpc/src/v1/traits/traces.rs

```diff
@@ -35,6 +35,9 @@ pub trait Traces: Sized + Send + Sync + 'static {
 	/// Executes the given call and returns a number of possible traces for it.
 	fn call(&self, _: Params) -> Result<Value, Error>;
 
+	/// Executes the given raw transaction and returns a number of possible traces for it.
+	fn send_raw_transaction(&self, _: Params) -> Result<Value, Error>;
```

Comment:
It does not actually send the transaction, does it? Perhaps `trace_raw_transaction` would be a better name